README.md

```diff
@@ -4,35 +4,28 @@
 
 This is a Calendar Picker Component for React Native
 
+### Migrated from moment to date-fns - New in 8.x
```

Comment:
Recommend changing this to:

> Breaking changes in 8.x - replaced moment with date-fns

And in the body mention that users who want to continue to use Moment should stick with 7.x

Reply:
I've updated the README as per your suggestions, and re-added the Prerequisites section.